e2e-tests/README.md

```diff
@@ -201,6 +201,19 @@ PWDEBUG=console yarn playwright test --headed --config admin/playwright.config.j
 PWDEBUG=console yarn playwright test --headed --config admin/playwright.config.js login.spec.js --debug
 ```
 
+##### Admin (Using a Local Branch)
+
+```shell
+# In one terminal, run the `test_e2e_env.sh` script, and then do the following...
+cd boundary-ui/
+ENABLE_MIRAGE=false API_HOST=$BOUNDARY_ADDR yarn workspace admin start
```

Comment:
It might also be good to mention right now in dev we default to ent so we might need to make another change in the future to handle switching the default version since it was expected a user would manually change the edition while testing

Reply:
Is there a variable we can pass in to set it to a certain edition?

Reply:
Ah I was saying rn I _think_ there might need to be a code change in our config to handle taking in an env variable, I don't think we expected to need to have the default value be set to something as the expectation was we would use the picker, this could be a separate PR